docs/setup/settings.asciidoc

```diff
@@ -29,6 +29,27 @@ service that Kibana uses to display map tiles in tilemap visualizations. By defa
 `tilemap.options.attribution:`:: *Default: `"© [Elastic Tile Service](https://www.elastic.co/elastic-tile-service)"`* The map attribution string.
 `tilemap.options.subdomains:`:: An array of subdomains used by the tile service.
 Specify the position of the subdomain the URL with the token `{s}`.
+
+[[regionmap-settings]] `regionmap`:: configuration object to include additional vector layers for use in the <<regionmap,Region Map visualization>>.
```

Comment:
"configuration object" should have a capital letter. "Configuration object". All other config descriptions seem to.

<img width="789" alt="screen shot 2017-06-29 at 5 54 20 pm" src="https://user-images.githubusercontent.com/25181823/27712224-ff25e200-5cf3-11e7-8759-13eea4d3c735.png">